README.md

```diff
@@ -11,6 +16,7 @@ Since these steps were parsed from a few different sources, I thought it would b
 5. ran ` ember install ember-cli-typescript@latest`
 6. ran `ember g component -gc welcome` (**note:** _see below if your confused by the output_)
 7. Add eslint/prettier or any further tsconfig if needed
+8. For better accessibility, add your preferred lang via the lang attribute in the html tag inside `app/index.html` (e.g. `<html lang="en">`) (_**note:** happens automatically when using `ember new` (referenced here:_ <a href="https://guides.emberjs.com/release/tutorial/part-1/orientation/#toc_creating-a-new-ember-app-with-ember-cli" target="_blank" rel="noopener noreferrer">#toc_creating-a-new-ember-app-with-ember-cli</a>))
```

Comment:
There is value in doing some of this stuff manually. That's personally how I learn best. Then once you know it, automate the heck out of it 😎